index.js

```diff
@@ -8,32 +8,44 @@ function sse(req, res, next) {
   res.status(200);
 
   // export a function to send server-side-events
-  res.sse = function sse(string) {
+  function send(string) {
     res.write(string);
-
     // support running within the compression middleware
     if (res.flush && string.match(/\n\n$/)) {
       res.flush();
     }
-  };
-  res.sseData = function sseData(data) {
-    res.sse('data: ' + JSON.stringify(data) + '\n\n');
-  this.isReady.reject();
+  }
+
+  function event(event, data) {
+    send('event: ' + event + '\n');
+    send('data: ' + JSON.stringify(data) + '\n\n');
```

Comment:
What about http://devdocs.io/node/os#os_os_eol?

Reply:
https://html.spec.whatwg.org/multipage/server-sent-events.html#parsing-an-event-stream

The spec doesn't require us to use the correct line break character or characters for the OS, it allows any of them interchangeably, so it's easier to just use `\n\n`.

Reply:
Nice and detailed! :+1: